Sources/Hummingbird/Middleware/MetricsMiddleware.swift

```diff
@@ -31,12 +31,12 @@ public struct MetricsMiddleware<Context: RequestContext>: RouterMiddleware {
                 // need to create dimensions once request has been responded to ensure
                 // we have the correct endpoint path
                 let dimensions: [(String, String)] = [
-                    ("hb_uri", context.endpointPath ?? request.uri.path),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", context.endpointPath ?? request.uri.path),
+                    ("method", request.method.rawValue),
                 ]
-                Counter(label: "hb_requests", dimensions: dimensions).increment()
+                Counter(label: "hb.requests", dimensions: dimensions).increment()
                 Metrics.Timer(
-                    label: "hb_request_duration",
+                    label: "hb.request_duration",
```

Comment:
should request_duration be named `hb.request_duration_seconds`?

Reply:
going with the OTel version